backend/src/handlers/helpful_links_handler.go

```diff
@@ -0,0 +1,134 @@
+package handlers
+
+import (
+	"UnlockEdv2/src/models"
+	"encoding/json"
+	"net/http"
+	"strconv"
+)
+
+func (srv *Server) registerLeftMenuRoutes() []routeDef {
+	axx := models.Feature()
+	return []routeDef{
+		{"GET /api/helpful-links", srv.handleGetHelpfulLinks, false, axx},
+		{"PUT /api/helpful-links", srv.handleAddHelpfulLink, true, axx},
+		{"PATCH /api/helpful-links/{id}/edit", srv.handleEditLink, true, axx},
+		{"PUT /api/helpful-links/toggle/{id}", srv.handleToggleVisibilityStatus, true, axx},
+		{"DELETE /api/helpful-links/{id}", srv.handleDeleteLink, true, axx},
+		{"PUT /api/helpful-links/activity/{id}", srv.handleAddUserActivity, false, axx},
+		{"PUT /api/helpful-links/sort", srv.changeSortOrder, true, axx},
+	}
+}
+
+var HelpfulSortOrder = make(map[uint]string)
+
+func (srv *Server) changeSortOrder(w http.ResponseWriter, r *http.Request, log sLog) error {
```

Comment:
I thought we were going to change the sort order whenever an administrator changed it? because it has to send a get request anyway to refresh the data, if they are an admin, we can just store the sort order of the last admin request sent.